tests/Clients/BaseRestResourceTest.php

```diff
@@ -63,6 +63,24 @@ public function testFindsWithParam()
         $this->assertEquals([1, "attribute"], [$resource->id, $resource->attribute]);
     }
 
+    public function testCanUseFalseyParams()
```

Comment:
```suggestion
    public function testFindsWithFalseyParams()
```
To keep the test naming conventions the same.